models/personel.py

```diff
@@ -0,0 +1,21 @@
+class Personel(object):
+    """personel model"""
+
+    def __init__(self, name):
+        self.name = name
+        want_accommodation = 'N'
```

Comment:
**Code Quality: Nitpick**

I presume this is an instance variable. You should use `self.wants_accommodation` if that's the case.